Lib/fontgoggles/mac/mainWindow.py

```diff
@@ -287,19 +287,35 @@ def setupSidebarGroup(self):
         group.feaVarTabs = self.feaVarTabs
         group.feaVarTabs.set(feaVarTabValues.index(self.project.uiSettings.feaVarTabSelection))
 
+        # Sidebar first tab
         featuresTab = group.feaVarTabs[0]
         self.featuresGroup = FeatureTagGroup(sidebarWidth - 6, {}, callback=self.featuresChanged)
         featuresTab.main = AligningScrollView((0, 0, 0, 0), self.featuresGroup, drawBackground=False,
                                               hasHorizontalScroller=False, autohidesScrollers=True,
                                               borderType=AppKit.NSNoBorder)
 
+        # Sidebar second tab
         variationsTab = group.feaVarTabs[1]
-        self.variationsGroup = SliderGroup(sidebarWidth - 6, {}, callback=self.varLocationChanged,
-                                           showHiddenAxes=self.project.uiSettings.showHiddenAxes)
+        self.variationsGroup = Group((0, 0, 0, 0))
```

Comment:
I couldn't really get this to work like this because SliderGroup seemed to always place its nested items absolutely. I refactored how `LabledView` passes its `posSize` to child elements and amended this where it is used elsewhere (the top of the sidebar Orientation/Script/Language labels) — then I was able to properly offset the `variationsGroup` as intended.